extensions/reactive-streams-operators/mutiny-reactive-streams-operators/deployment/src/main/java/io/quarkus/mutiny/reactive/operators/deployment/MutinyReactiveStreamsOperatorsProcessor.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.mutiny.reactive.operators.deployment;
+
+import org.eclipse.microprofile.reactive.streams.operators.ReactiveStreamsFactory;
+import org.eclipse.microprofile.reactive.streams.operators.core.ReactiveStreamsFactoryImpl;
+import org.eclipse.microprofile.reactive.streams.operators.spi.ReactiveStreamsEngine;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ServiceProviderBuildItem;
+import io.smallrye.mutiny.streams.Engine;
+
+public class MutinyReactiveStreamsOperatorsProcessor {
+
+    @BuildStep
+    public void build(BuildProducer<ServiceProviderBuildItem> serviceProvider,
+            BuildProducer<FeatureBuildItem> feature) {
+        feature.produce(new FeatureBuildItem(FeatureBuildItem.MUTINY_REACTIVE_STREAMS_OPERATORS));
```

Comment:
I'm not sure I would add a feature if it's a purely internal extension?

Reply:
I agree it was more something for me to check that it was the right implementation. 
Let me remove this.